packages/react-components/src/DocsComponents/FluentDocsPage.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import * as React from 'react';
+import {
+  DocsContext,
+  ArgsTable,
+  Title,
+  Subtitle,
+  Description,
+  HeaderMdx,
+  Primary,
+  PRIMARY_STORY,
+  Stories,
+} from '@storybook/addon-docs';
+
+import { Toc, idToHash } from './Toc.stories';
+
+const CONTAINER_WIDTH = '900px';
+const TOC_WIDTH = '300px';
+
+export const FluentDocsPage = () => {
+  const context = React.useContext(DocsContext);
+  const stories = context.storyStore.getStoriesForKind(context.kind);
+  const primaryStory = stories[0];
```

Comment:
I assume there's no way for stories to be empty?  If so, then you'll need to conditionally render primaryStory.